pkg/agent/openflow/pipeline.go

```diff
@@ -1469,14 +1472,31 @@ func (c *client) defaultDropFlow(tableID binding.TableIDType, matchKey *types.Ma
 		Done()
 }
 
-// localProbeFlow generates the flow to forward packets to conntrackCommitTable. The packets are sent from Node to probe the liveness/readiness of local Pods.
+// localProbeFlow generates the flow to forward locally generated packets to conntrackCommitTable, bypassing ingress
+// rules of Network Policies. The packets are sent by kubelet to probe the liveness/readiness of local Pods.
+// On Linux and when OVS kernel datapath is used, it identifies locally generated packets by matching the
+// HostLocalSourceMark, otherwise it matches the source IP. The difference is because:
+// 1. On Windows, kube-proxy userspace mode is used, currently there is no way to distinguish kubelet generated traffic
+//    from kube-proxy proxied traffic.
+// 2. pkt_mark field is not properly support for OVS userspace (netdev) datapath.
+// Note that there is a defect in the latter way that external clients that access NodePort service will masquerade as a
```

Comment:
How about: NodePort Service access by external clients will be masqueraded as...

Reply:
Done, thanks